amplify/data/resource.ts

```diff
@@ -8,6 +8,8 @@ const SessionStorageModel = {
   templateType: a.enum([...templateTypes, 'UNKNOWN']),
   nhsAppTemplateName: a.string().required(),
   nhsAppTemplateMessage: a.string().required(),
+  smsTemplateName: a.string(),
```

Comment:
should the `nhsAppTemplateName` and `nhsAppTemplateMessage` still be required?

Reply:
I agree. They shouldn't be required. But I don't think it should be changed as part of this PR.